test/unit/options/normalizers_test.rb

```diff
@@ -429,15 +439,45 @@
     end
 
     it "should accept and parse iso8601" do
-      _(formatter.format("1986-01-01T08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01T08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      end
     end
 
     it "should accept and parse YYYY-MM-DD HH:MM:SS" do
-      _(formatter.format("1986-01-01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      end
     end
 
     it "should accept and parse YYYY/MM/DD HH:MM:SS" do
-      _(formatter.format("1986/01/01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986/01/01 08:30:20")).must_equal("1986-01-01T08:30:20+00:00")
+      end
+    end
+
+    it "should parse iso8601 compliant timezones" do
+      mock_env("TZ" => "UTC") do
+        _(formatter.format("1986-01-01 08:30:20 +04:00")).must_equal("1986-01-01T08:30:20+04:00")
+      end
+    end
+
+    it "should not recognize invalid timezones" do
```

Comment:
The name is slightly misleading as it also ignores "valid" timezones such as `EDT`.

Reply:
What name would you suggest? `should not recognize certain timezones` because it recognises some.

Reply:
maybe something along the lines of `should not recognize timezone strings not formatted according to iso8601`?

Reply:
That is also slightly misleading because it recognizes timezones like `EST`